### PR TITLE
[Term Entry]: PHP stripos()

### LINKED ISSUE #2014: [Term Entry] PHP String Functions stripos()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `stripos()` function.

The entry should go in a new file in docs/content/php/concepts/string-functions/terms/stripos/stripos.md

The entry should

- Have a syntax section illustrating the function with a ` ```pseudo ` code block.
- An example showing the function in use with a ` ```php ` code block.
- A runnable codebyte example in a ` ```codebyte/php` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Jess-G95**
### Description

Added a new entry for the PHP function stripos(). The entry is in a new file in docs/content/php/concepts/string-functions/terms/stripos/stripos.md

Closes #2014 

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [reply content/php/concepts/string-functions/terms/stripos/stripos.md:35]: @dakshdeepHERE, @Jess-G95 Maintainer here. Subheads under a "##" header are allowed, and we do not use boldface for headers.

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [reply content/php/concepts/string-functions/terms/stripos/stripos.md:39]: @dakshdeepHERE, @Jess-G95 Maintainer here. Subheads under a "##" header are allowed, and we do not use boldface for headers.

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [reply content/php/concepts/string-functions/terms/stripos/stripos.md:67]: @dakshdeepHERE, @Jess-G95 Maintainer here. Subheads under a "##" header are allowed, and we do not use boldface for headers.

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [reply content/php/concepts/string-functions/terms/stripos/stripos.md:53]: @dakshdeepHERE, @Jess-G95 Maintainer here. Subheads under a "##" header are allowed, and we do not use boldface for headers.

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/stripos/stripos.md:35]:
```suggestion
Required Parameters:

- `given_string` is the string that the function searches.
- `string_to_find` is the string to search within `given_string`.

Optional Parameters:

- `start_search` tells the function to start the search at a specified index.
```

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/stripos/stripos.md:80]:
```suggestion
```
We don't use output for codebytes. The user can run them and see the output themselves.

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/stripos/stripos.md:39]:
```suggestion
### With Required Parameters
```

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (SSwiniarski) [inline content/php/concepts/string-functions/terms/stripos/stripos.md:53]:
```suggestion
### With Required and Optional Parameters
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Jess-G95 I did a first maintainer review. Tag me when you've addressed my comments and @dakshdeepHERE's unresolved comment.

**Comment by Jess-G95** [comment]
@SSwiniarski thank you. I have addressed your comments and @dakshdeepHERE's comment and committed my changes.

**Comment by dakshdeepHERE** [comment]
> @SSwiniarski thank you. I have addressed your comments and @dakshdeepHERE's comment and committed my changes. 

🚀

**Review by SSwiniarski — APPROVED** [review]
@Jess-G95 Looks good for a second review.

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (caupolicandiaz) [inline content/php/concepts/string-functions/terms/stripos/stripos.md:63]: Ideally each example in an entry is demonstrating some new functionality. In other words, we strive to add value with each example and not include examples that just duplicate functionality shown elsewhere in the entry. I would suggest making the second example the codebyte and cutting the 'pizza' example, which is redundant.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@Jess-G95 Just added one comment. Please update, and tag me when you've made the changes. Thanks

File: content/php/concepts/string-functions/terms/stripos/stripos.md
Comment (Jess-G95) [reply content/php/concepts/string-functions/terms/stripos/stripos.md:63]: Ok, noted. I have removed the 'pizza' codebyte example and made the second example the codebyte instead. Is it ok to have an example that doesn't return anything in the output when the user runs the codebyte?  If not, I can change the index to start the search, or the string to find, so there will be an output.

Comment (caupolicandiaz) [reply content/php/concepts/string-functions/terms/stripos/stripos.md:53]: Yes, I think it's preferable to have an output. I've modified the example further to provide users with some options.

**Review by caupolicandiaz — APPROVED** [review]
@Jess-G95 @SSwiniarski I've made a few small changes, looks good to merge.

**Comment by SSwiniarski** [comment]
@Jess-G95 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/string-functions/stripos

image

**Pull request merged**
